logstash-core/src/main/java/org/logstash/execution/WorkerLoop.java

```diff
@@ -65,7 +60,7 @@ public WorkerLoop(
     {
         this.consumedCounter = consumedCounter;
         this.filteredCounter = filteredCounter;
-        this.execution = pipeline.buildExecution();
+        this.execution =  pipeline.buildExecution(preserveEventOrder);
```

Comment:
nitpick: extra space

```suggestion
        this.execution = pipeline.buildExecution(preserveEventOrder);
```